app/controllers/PostProcess.scala

```diff
@@ -98,7 +98,7 @@ object PostProcess extends JSONParser {
     toSimpleVertexArrJson(queryResultLs, Seq.empty[QueryResult])
   }
 
-  def toSimpleVertexArrJson(queryResultLs: Seq[QueryResult], exclude: Seq[QueryResult]): JsValue = {
+  def toSimpleVertexArrJson(queryResultLs: Seq[QueryResult], exclude: Seq[QueryResult], limit: Int = Query.defaultLimit): JsValue = {
```

Comment:
@SteamShon 
Actually, I agree with that using global limit clause from Query instance.
As you mentioned, the limit clause can be used from QueryResult as below..
```
val q = queryResultLs.head.query
take(q.limit)
```
I think its better to discuss in depth about it at new issue, if you have plan to refactor QueryResult.